pyembed/src/importer.rs

```diff
@@ -51,27 +57,26 @@ type py_init_fn = extern "C" fn() -> *mut oldpyffi::PyObject;
 #[cfg(windows)]
 fn extension_module_shared_library_create_module(
     resources_state: &PythonResourcesState<u8>,
-    py: cpython::Python,
-    sys_modules: cpython::PyObject,
-    spec: &cpython::PyObject,
-    name_py: cpython::PyObject,
+    py: Python,
+    sys_modules: &PyAny,
+    spec: &PyAny,
+    name_py: &PyAny,
     name: &str,
     library_data: &[u8],
-) -> cpython::PyResult<cpython::PyObject> {
-    let origin = cpython::PyString::new(py, "memory");
+) -> PyResult<PyObject> {
+    let origin = PyString::new(py, "memory");
 
-    let existing_module = unsafe {
-        oldpyffi::_PyImport_FindExtensionObject(name_py.as_ptr(), origin.as_object().as_ptr())
-    };
+    let existing_module =
+        unsafe { pyffi::_PyImport_FindExtensionObject(name_py.as_ptr(), origin.as_ptr()) };
 
     // We found an existing module object. Return it.
     if !existing_module.is_null() {
-        return Ok(unsafe { cpython::PyObject::from_owned_ptr(py, existing_module) });
+        return Ok(unsafe { PyObject::from_owned_ptr(py, existing_module) });
     }
 
     // An error occurred calling _PyImport_FindExtensionObjectEx(). Raise it.
-    if !unsafe { oldpyffi::PyErr_Occurred() }.is_null() {
-        return Err(cpython::PyErr::fetch(py));
+    if !unsafe { pyffi::PyErr_Occurred() }.is_null() {
+        return Err(PyErr::fetch(py).expect("Python exception should be set"));
```

Comment:
Interesting to see this. Makes me think `PyErr::api_call_failed` may want to be a public API of PyO3?